hwconf/shutdown.c

```diff
@@ -67,7 +68,11 @@ void shutdown_set_sampling_disabled(bool disabled) {
 }
 
 static bool do_shutdown(void) {
-	conf_general_store_backup_data();
+	// Store data if 1000m or more have been done to possibly reduce flash
```

Comment:
I think it is safe to assume that writing the backup date is fine when a shutdown is triggered. Otherwise a lot of odometer data can remain unlogged.

Reply:
the amount of **_possible_** odometer data unlogged can be reduced by the distance, even 1m could be used even if it's  pretty useless as not even car odometers have such accuracy, say one could set 10m or 100m.
Anyway it was just a suggestion as we don't use external eeprom like in automotive odometers.

Reply:
Yes, I think 1000m is fine for accuracy, we do not need the exact metres. It may shift over time, but from someone who does not have ANY mileage data, this is a very good start. The only thing that worries me is the flash enurance of the F405, could this become a problem?